### ISSUE TITLE
About the same padding

### ISSUE THREAD

**Issue by ckqsars** [zylo117/Yet-Another-EfficientDet-Pytorch#50]
I find that the conv2d in pytorch has the padding, the description is 
padding(int or tuple, optional): Zero-padding added to both sides of the input. Default: 0

Is it same as the same padding  in tensorflow?

**Comment by zylo117** [comment]
No, different strategies

**Comment by ckqsars** [comment]
Could you tell me the difference between the same padding  and the conv2d(padding=1). 
As I know,  both of they pad the image with zero.

**Comment by zylo117** [comment]
different strategies.
For examples, I want to pad 3 more data on the both sides of a 1x5 array.
Whay should I do? 2 on the left or 1 on the right? or the other way around?

That's padding strategies, and there isn't one in pytorch. So I have to manually pad them.
